src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,155 @@
+<template>
```

Comment:
This is an exact copy of the shared component, so I can develop it as needed locally, and you can look at subsequent commits to see any changes I make to it.

In the end, this will either stay as a local component (if it's specific to Filings UI), or I'll move it back to [bcrs-shared-components](https://github.com/bcgov/bcrs-shared-components).